Rework a bit the May content for step 3 (#623)
akeneo/pim-helpcenter: content/updates/2020-05/1-easily-monitor-errors-on-your-connections.md

# Easily monitor errors on your connections 
::: meta-data type="New" features="Connections, Productivity" available="mid-May" link-to-doc="../articles/connection-dashboard.html#in-the-connection-monitoring-page"
:::

> The new `Monitoring` page available under `Connections` lists the last errors for your Connections. From this page, you can now easily track and fix any error on product creation, update, or deletion for each of your connections. 

Author: ```suggestion
For each of your connections, a new `Monitoring` page now lists the last integration errors that may have occurred.

From this page, you can now easily track and fix any error on product creation, update, or deletion for each of your connections. 
```
